Gems/Atom/RPI/Code/Source/RPI.Public/View.cpp

```diff
@@ -276,11 +308,25 @@ namespace AZ
                         SortDrawList(drawListsByTag[idx], RHI::DrawListTag(idx));
                     };
                     Job* jobSortDrawList = aznew JobFunction<decltype(jobLambda)>(jobLambda, true, nullptr); // Auto-deletes
-                    jobSortDrawList->SetDependent(&jobCompletion);
-                    jobSortDrawList->Start();
+                    if (parentJob)
+                    {
+                        parentJob->StartAsChild(jobSortDrawList);
+                    }
+                    else
+                    {
+                        jobSortDrawList->SetDependent(&jobCompletion);
+                        jobSortDrawList->Start();
+                    }
                 }
             }
-            jobCompletion.StartAndWaitForCompletion();
+            if (!parentJob)
```

Comment:
Same as previous comment - minor suggestion to switch to if (parentJob).

Reply:
re-ordered